src/main.js

```diff
@@ -122,83 +115,77 @@ var quotes = [
 var savedPosters = [];
 var currentPoster;
 
-// event listeners go here 👇
 window.addEventListener('load', generatePoster);
 randomPosterBtn.addEventListener('click', generatePoster);
 makePosterBtn.addEventListener('click', makePoster);
```

Comment:
Naming nitpick - I might switch `makePoster` for `createCustomPoster`, to give more clarity to the functionality